webpack-config/webpack.config.js

```diff
@@ -0,0 +1,3 @@
+const config = require( '@wordpress/scripts/config/webpack.config.js' );
```

Comment:
Next step, get rid of this file altogether :)

Reply:
yay! https://github.com/WordPress/gutenberg/pull/13877